packages/x-bundle/src/validators/declarations.ts

```diff
@@ -21,10 +21,10 @@ declare module "yup" {
      */
     uniqueField(config?: IUniqueFieldValidationConfig): NumberSchema;
   }
-
+  //@ts-ignore
```

Comment:
why?

Reply:
had some type error blocking my unit testing in my loca, just forgot not to delete it XD